src/third-party/PhishingController.ts

```diff
@@ -182,6 +202,7 @@ export class PhishingController extends BaseController<
    * Updates lists of approved and unapproved website origins.
    */
   async updatePhishingLists() {
+    this.lastFetched = Date.now();
```

Comment:
Should this go at the end of the method rather than the beginning, to wait until after all requests are finished?